packages/chrome/src/styled/sheet/StyledSheet.ts

```diff
@@ -18,26 +23,42 @@ interface IStyledSheetProps {
   size?: string;
 }
 
-const borderStyle = ({
+const colorStyles = ({ theme, isOpen }: IStyledSheetProps & ThemeProps<DefaultTheme>) => {
+  const backgroundColor = getColor({ theme, variable: 'background.raised' });
```

Comment:
i would say Sheet's background would be `default`, as it is supplementary view alongside main content.

Reply:
@steue yeah, agreed. Do we have corresponding designs for the `Sheet` component? Also, are the subtle borders still correct with the default background?

Reply:
@jzempel @'d you on Figma! 💪🏻 good catch on the border treatment. like modals and pane, surrounding border is `default`, with the header/footer using `subtle`.